test/e2e/internal/utils/testdata.go

```diff
@@ -16,10 +16,14 @@ limitations under the License.
 package utils
 
 const (
-	PreviewDesc  = "** This command is in preview and under development. **"
-	ExampleDesc  = "\nExample - "
-	ImageRepo    = "command/images"
-	ArtifactRepo = "command/artifacts"
-	Repo         = "command/images"
-	Namespace    = "command"
+	PreviewDesc       = "** This command is in preview and under development. **"
+	ExampleDesc       = "\nExample - "
+	ImageRepo         = "command/images"
+	ArtifactRepo      = "command/artifacts"
+	Repo              = "command/images"
+	Namespace         = "command"
+	FoobarHeaderInput = "Foo:bar"
+	FoobarHeader      = "\"Foo\": \"bar\""
+	AbHeaderInput     = "A: b"
+	AbHeader          = "\"A\": \"b\""
```

Comment:
Those constants can be moved to `custom_header_test.go`

Reply:
removed and moved to the applicable function.